web_tool_script_3.R

```diff
@@ -169,14 +168,61 @@ if (file.exists(file.path(results_path, portfolio_name, "Bonds_results_map.rda")
     "scenario_geography" = NA_character_
   )
 }
-# bonds_results_map <-read_rds(file.path(results_path, portfolio_name, paste0(portfolio_name, "_Bonds_results_map.rda")))
 
-# bonds_results_map <- equity_results_map %>% filter(portfolio_name == "")
+# load equity stress test data
+if (file.exists(file.path(results_path, portfolio_name, "Equity_results_stress_test.rda"))) {
+  equity_results_stress_test <- read_rds(file.path(results_path, portfolio_name, "Equity_results_stress_test.rda"))
+} else {
+  equity_results_stress_test <- tibble(
```

Comment:
Nit: It'd be nice for each script to start with expressive high level code that summarizes what the rest of the script does.
```
if (some_expressive_condition()) {
  some_expressive_wrapper()
} else {
 some_other_expressive_wrapper()
}
```

Reply:
related to this: https://github.com/2DegreesInvesting/PACTA_analysis/issues/213